src/deluge/modulation/params/param.h

```diff
@@ -197,8 +197,10 @@ enum UnpatchedGlobal : ParamType {
 	UNPATCHED_PAN,
 	UNPATCHED_LPF_FREQ,
 	UNPATCHED_LPF_RES,
+	UNPATCHED_LPF_MORPH,
```

Comment:
need to add these new params to the unpatchedNonGlobalParams array below (lines 322-325)